CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing to The Sailscasts Docs
+
+We appreciate your interest in contributing to this docs! Your contributions are very valuable to us, and we want to ensure that the process is easy and productive for everyone involved. 
+
+Please take a moment to read the guidelines below before contributing to the project.
+
+## Project Description
+
+This project contains the official documentation of all the products built for The Sailscasts Community.
+
+#### These products includes:
+- [boring-stack](https://github.com/sailscastshq/boring-stack)
+- [create-sails](https://github.com/sailscastshq/create-sails)
+- [inertia-sails](https://github.com/sailscastshq/inertia-sails)
+- [wish](https://github.com/sailscastshq/sails-hook-wish)
+- [captain-vane](https://github.com/sailscastshq/captain-vane)
+- [guppy](https://guppy.sailscasts.com/)
+
+
+## Contribution Workflow
+
+When contributing to this project, we adhere to the [Gitflow branching model](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow). 
+
+In this workflow, we use the develop branch as the central branch for active development. Therefore, we request that all Pull Requests be targeted towards the develop branch instead of the main branch. This approach ensures the stability of our main branch while allowing continuous development on the develop branch.
+
+## Before Contributing
+
+Before diving into the contribution process and setting up your environment, it is essential to select an issue you would like to work on.
+
+#### Here's a simple guide to follow:
+
+1. Search through the list of [open issues](https://github.com/sailscastshq/docs.sailscasts.com/issues).
```

Comment:
Nice, do you think we should mention the possibility of creating an issue if one doesn't exist for the PR you want to submit?

Reply:
I mentioned that in number 2 of "Before Contributing"